kong/templates/nginx_kong.lua

```diff
@@ -303,6 +303,12 @@ server {
     access_log ${{STATUS_ACCESS_LOG}};
     error_log  ${{STATUS_ERROR_LOG}} ${{LOG_LEVEL}};
 
+> if status_ssl_enabled then
+    ssl_certificate     ${{STATUS_SSL_CERT}};
+    ssl_certificate_key ${{STATUS_SSL_CERT_KEY}};
+    ssl_session_cache   shared:StatusSSL:10m;
```

Comment:
This can be set to `1m` to reduce memory usage, since there won't be that many clients for this API.

Reply:
I changed it, but memory usage is not when you start Nginx, but when you actually use it.